oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/FormFeedback/FormFeedback.jsx

```diff
@@ -20,21 +20,33 @@ const CustomMessage = ({ children, ...uiProps }) => {
 };
 export const FormFeedback = () => {
   const { values } = useFormikContext();
-  const validationErrors = getIn(values, "validationErrors", {});
+  const BEvalidationErrors = getIn(values, "BEvalidationErrors", {});
+  const FEvalidationErrors = getIn(values, "FEvalidationErrors", {});
```

Comment:
Can you un-capitalize this please (to avoid confusion with a React component name)?